.env.example

```diff
@@ -1,3 +1,5 @@
 DESCOPE_PROJECT_ID= # Your Descope Project ID
 DESCOPE_FLOW_ID="sign-up-or-in" # Your Descope flow ID
 DESCOPE_FLOW_DEBUG= # Set to true in case you want to debug your flow
+REACT_APP_DESCOPE_BASE_URL= # Descope API base URL
+REACT_APP_ORIGIN_BASE_URL= # Set in case you want to use the origin as
```

Comment:
the name should indicate that it is a boolean (`REACT_APP_USE_ORIGIN_BASE_URL`)

also, I would avoid using origin, and prefer setting up explicitly the proper confs in localhost , but not going to fight it

Reply:
it can't be a config, it needs to be evaluated at runtime

Reply:
can you plz explain why